Xamarin.Forms.Maps.UWP/MapRenderer.cs

```diff
@@ -174,6 +177,8 @@ void LoadPin(Pin pin)
 						LoadUserPosition(userPosition.Coordinate, moveToLocation);
```

Comment:
The above code (using myGeolocator) crashes if the location is not enabled. LocationStatus returns PositionStatus.NotInitialized.